app/controllers/shared/api.php

```diff
@@ -74,17 +74,20 @@
         ->from($queueForEvents)
         ->trigger();
 
-    $queueForWebhooks
-        ->from($queueForEvents)
-        ->trigger();
 
-    if ($queueForEvents->getProject()->getId() === 'console') {
-        return;
+    /** Trigger webhooks events only if a project has them enabled */
+    if ($project->getAttribute('webhooks', []) !== []) {
```

Comment:
Are you sure this condition works? wouldn't something like array !empty() would be safer here?

Reply:
Fixed 👍